### PR TITLE
discarding staticATokens from doc

### PR DESCRIPTION
**Pull request by hsouf**

### CODE REVIEW

**Comment by achab** [comment]
General remarks about the README: 
1. The "Installation" section should be above the "Documentation" part
2. Could we add a table of content?
3. On the top of the file, tags' numbers (pull requests and issues) don't seem to match the real numbers

File: README.md [hunk 1]
```diff
@@ -42,32 +41,32 @@ The bridge is also shaped for liquidity providers who are able to assume Ethereu
...
 
-## Bridging staticATokens from L1<>L2
+## Bridging aTokens from L1<>L2
 
 ### Transfer L1->L2: 
 
 
-Users can either bridge their staticAToken to L2 by calling `deposit()` on `TokenBridge`, or deposit the underlying asset (let's say Dai) or the corresponding aToken (i.e. aDai) by calling `depositUnderlying()`.
+Users can either bridge their aToken (let's say aDai) to L2 by calling `deposit()` on `TokenBridge`, or deposit the underlying asset (i.e Dai).
+Users wil then have to approve the bridge to spend the underlying `asset` tokens or `aTokens`, depending on the provided value for `fromAsset` argument when depositing.
```
Comment (JorikSchellekens) [inline README.md:52]:
```suggestion
Users will have to approve the bridge to spend the underlying `asset` tokens or `aTokens`, depending on the provided value for `fromAsset` argument when depositing.
```

File: README.md [hunk 2]
```diff
@@ -42,32 +41,32 @@ The bridge is also shaped for liquidity providers who are able to assume Ethereu
...
-Calling `depositUnderlying` will result in the following:
+If depositing underlying `asset`:
 
-Users will have to approve the bridge to spend the underlying `asset` tokens or `aTokens`, depending on the provided value for `fromAsset` argument when depositing.
-
-- The underlying asset of the staticAToken or the aToken (depending on the user's input )  will be transfered from the user account to the L1 bridge.
-- The bridge will then deposit the aTokens/asset on the staticAToken.
-- New staticATokens will be minted to the bridge.
+- The `asset` token will be transfered from the user account to the L1 bridge.
+- The bridge will then deposit the `asset` token on the aToken.
```
Comment (JorikSchellekens) [inline README.md:60]:
```suggestion
- The bridge will then deposit the `asset` token in the aToken.
```

File: README.md [hunk 3]
```diff
@@ -42,32 +41,32 @@ The bridge is also shaped for liquidity providers who are able to assume Ethereu
...
+- New aTokens will be minted to the bridge.
```
Comment (JorikSchellekens) [inline README.md:84]: I'd drop this line

File: README.md [hunk 4]
```diff
@@ -42,32 +41,32 @@ The bridge is also shaped for liquidity providers who are able to assume Ethereu
...
 - A message will be sent to the  L2 bridge with the amount to be transferred, the L1 token address and the recipient address as parameters.
```
Comment (JorikSchellekens) [inline README.md:85]:
```suggestion
- A message will be sent to the  L2 bridge with the amount of `ETHstaticAToken` to be transferred, the L1 token address and the recipient address as parameters.
```

File: README.md [hunk 5]
```diff
@@ -2,29 +2,46 @@
...
-[![Issues](https://img.shields.io/github/issues-raw/tterb/PlayMusic.svg?maxAge=25000)](https://github.com/aave-starknet-project/aave-starknet-bridge/issues)
-
- <a href="https://github.com/aave-starknet-project/aave-starknet-bridge/blob/main/LICENSE.md"><img src="https://img.shields.io/github/license/sourcerer-io/hall-of-fame.svg?colorB=ff0000"></a>
+[![Tests](https://github.com/aave-starknet-project/aave-starknet-bridge/actions/workflows/deploy.yml/badge.svg)](https://github.com/aave-starknet-project/aave-starknet-bridge/actions/workflows/deploy.yml)
+<a href="https://github.com/aave-starknet-project/aave-starknet-bridge/blob/main/LICENSE.md"><img src="https://img.shields.io/github/license/sourcerer-io/hall-of-fame.svg?colorB=ff0000"></a>
 
 :warning: This codebase is still in an experimental phase, has not been audited, might contain bugs and should not be used in production.
 
+
+## Table of contents
```
Comment (achab) [inline README.md:11]: Thanks a lot for this! We could add numbers and group some sections together, like this:

```
## Table of contents
  * [1. Overview](#overview)
  * [2. Architecture](#architecture)
    * [2.1. Contracts](#contracts)
    * [2.2. ETHStaticATokens on L2](#ethstaticatokens-on-l2)
    * [2.3. Bridging aTokens from L1<>L2](#bridging-atokens-from-l1--l2)
      * [2.3.1. Transfer L1->L2](#transfer-l1--l2-)
      * [2.3.2. Transfer L2->L1](#transfer-l2--l1-)
    * [2.4. Synchronisation of rewards on L1 <> L2](#synchronisation-of-rewards-on-l1----l2)
    * [2.5. Claiming rewards on L2](#claiming-rewards-on-l2)
    * [2.6. Bridging rewards from L2->L1](#bridging-rewards-from-l2--l1)
    * [2.7. Proxies](#proxies)
    * [2.8. Governance](#governance)
  * [3. Development](#governance)
    * [3.1 Setup](#environment)
    * [3.2 Tests](#run-the-tests)
    * [3.3. Deployment](#deployment)
```

File: README.md [hunk 6]
```diff
@@ -2,29 +2,46 @@
...
+  * [Deployment](#deployment)
+
+
+
 ## Overview
 
 For Aave, one of the main current and future goals is growth of liquidity and user base. As seen on side-chains with low transaction cost like Polygon or Avalanche, there is high demand to use the Aave protocol with small amounts to earn high yield. That's why we brought to you an initial phase of the Aave <> Starknet integration allowing deposit/withdrawal on Aave Ethereum by exclusively transacting on Starknet.
 
-The bridge allows users to deposit and withdraw `staticATokens` - wrappers converting balance-increasing [aTokens]( https://docs.aave.com/developers/tokens/atoken) into exchange-rate-increasing staticATokens - on StarkNet and get wrapped tokens `ETHStaticATokens` that allow users to keep enjoying the same rewards as on L1. 
+The bridge allows users to deposit and withdraw their `aTokens` on StarkNet and get wrapped tokens `ETHStaticATokens` that allow users to keep enjoying the same rewards as on L1.
```
Comment (achab) [inline README.md:34]: Why removing the link to `aToken definition? Also, you should add a small description for staticAToken. Readers coming to this repo are not all knowledgeable about Aave infra.

File: README.md [hunk 7]
```diff
@@ -2,29 +2,46 @@
...
 
-The bridge allows users to deposit and withdraw `staticATokens` - wrappers converting balance-increasing [aTokens]( https://docs.aave.com/developers/tokens/atoken) into exchange-rate-increasing staticATokens - on StarkNet and get wrapped tokens `ETHStaticATokens` that allow users to keep enjoying the same rewards as on L1. 
+The bridge allows users to deposit and withdraw their `aTokens` on StarkNet and get wrapped tokens `ETHStaticATokens` that allow users to keep enjoying the same rewards as on L1. 
 
 The bridge is also shaped for liquidity providers who are able to assume Ethereum cost of deposit/withdrawal, as they transact large amounts. They will deposit on Aave Ethereum, bridge the staticATokens to Starknet and make them available for users there to buy and hold, accruing this way yield from L1. 
 
 ## Architecture
+![aave_bridge_v2](https://user-images.githubusercontent.com/37840702/167132315-70923006-e5ef-42a8-a431-37bdb6c252c8.png)
```
Comment (achab) [inline README.md:39]: very good! the image could be added to the repo

File: README.md [hunk 8]
```diff
@@ -42,58 +59,64 @@ The bridge is also shaped for liquidity providers who are able to assume Ethereu
...
+Users can either bridge their aToken (let's say aDai) to L2 by calling `deposit()` on `TokenBridge`, or deposit the underlying asset (i.e Dai).
+Users wil have to approve the bridge to spend the underlying `asset` tokens or `aTokens`, depending on the provided value for `fromAsset` argument when depositing.
 
-Calling `deposit` will result in the following:
 
-- staticATokens will be transfered from the user account to the L1 bridge.
-- A `deposit` event will be emitted with L1 token address, L2 recipient address and the amount.
-- A message will be sent to the  L2 bridge with the amount to be transferred, the L1 token address and the L2 recipient address as parameters.
-- The token bridge on L2 will then mint the correspending ETHStaticAToken of the L1 token to the user.
+Calling `deposit` will result in the following:
```
Comment (achab) [inline README.md:73]: Calling `deposit` allows users deposit `aToken` or their underlying `asset`:

File: README.md (hunk 6 above)
Comment (hsouf) [reply README.md]: restored :)

File: README.md (hunk 5 above)
Comment (hsouf) [reply README.md:12]: agreed

File: README.md [hunk 9]
```diff
@@ -2,116 +2,144 @@
...
 
-To bridge their rewards to L1, users will have to call `bridge_rewards()` on the L2 `token_bridge` providing the `amount` to be bridged. The L1 bridge will be claiming pending rewards to self from all staticATokens in a loop until having enough rewards balance to transfer it to users. The claiming ans transferring of rewards is handled by `receiveRewards()` on `TokenBridge.sol`
 
 
 ## Proxies
 
 All calls made to the following contracts will be handled by a proxy who delegates the calls to the available implementation of these contracts.
 - Token bridge on L2 
-- ETHStaticATokens on L2
+- staticATokens on L2
```
Comment (JorikSchellekens) [inline README.md:130]: We'll be renaming these to `static_a_tokens`

Comment (JorikSchellekens) [reply README.md:130]: I'll rename this in a separate pr

**Review by JorikSchellekens — APPROVED** [review]

**Pull request merged**